File: <100/024/024.py
import itertools


def main() -> None:
    S: str = "0123456789"
    iters = itertools.permutations(S)
    for j, item in enumerate(iters):
        if j == 10**6 - 1:
            answer = ''.join(item)
            print(answer)
            return


if __name__ == '__main__':
    main()

File: <100/024/024.jl
using Combinatorics


function main()
  S:: String = "0123456789"
  perms = permutations(S, length(S))

  for (cnt, item) in enumerate(perms)
    if cnt == 10^6
      item |> String |> println
      return
    end
  end
end


@time main()
